.travis.yml

```diff
@@ -30,53 +30,46 @@ jobs:
   allow_failures:
     - name: "Snake Case (Optional)"
   include:
```

Comment:
Can you confirm that the snake case check is still optional in the pipeline. I can't seem to find the view i was on that i posted on slack were it explicitly said this job is optional. This job is required to be optional as the check outputs a lot of false positives when working with third party libraries. The flex i wrote for snake case check is a does not take into account the source of a token merely says if a token is snake case or not (hence the false positives).

Reply:
Judging by the documentation, ```allow_failures``` always refers to a job. And as the Snake Case test is a stage, this won't work. So unless I'm mistaken, you need to make Snake Case a separate job if you want it to be optional.

Reply:
When i had the allow failures in section in the main job includes i had that table output (slack) from Travis stating it was optional. If you can find this page showing the job is optional then this syntax is okay. Else it may need more testing (withing the main job list, where it is... etc)

Reply:
I found the page i'm talking about! Seems like the only way to access it is through Github in the checks section below (click details on the Travis tests). Comparing it to my screen shot the snake case currently is not optional. I'm assuming this can be fixed by moving allowed_failures back to the main job block?

Reply:
You're right, snake case is no longer optional for the changes in my PR. I'm off to a business trip, but I'll try to fix that once I get home.

Reply:
Snake case is now optional (again). See: https://github.com/KhronosGroup/Vulkan-Samples/pull/12/checks?check_run_id=308540944